### DISCUSSION TITLE
smaller q-fab element

### DISCUSSION THREAD

**Discussion by c0d1ngkjy** [quasarframework/quasar#15746] [General - Components / Directives / etc]
hello quasar
I am struggling with adjusting the size of q-fab element (I'm trying to make it smaller)
Can't find any reference in quasar docs

I tried giving it fixed height with style="height:50px"
Also tried giving it a parent div and giving the div fixed size and still doesn't work

**Comment by metalsadman** [comment]
https://codepen.io/metalsadman/pen/xxyqaZQ
